shared/chat/blocking/block-modal/index.tsx

```diff
@@ -59,7 +59,9 @@ const CheckboxRow = (props: CheckboxRowProps) => {
         />
         <Kb.Box style={styles.iconBox} />
         {props.info && !infoShowing && (
-          <Kb.Icon type="iconfont-question-mark" color="grey" onClick={() => setInfoShowing(true)} />
+          <Kb.WithTooltip tooltip={props.info}>
+            <Kb.Icon type="iconfont-question-mark" color="grey" onClick={() => setInfoShowing(true)} />
```

Comment:
should this still get an onClick event? Since we want the tooltip to show up on hover.

Reply:
I messaged you asking that. Sounds like the answer is no, will do.

Reply:
fixed 🔧